src/contentScript.js

```diff
@@ -137,13 +139,13 @@ var __gcrExtAnswers;
     var spanText = document.createElement('text');
     spanText.innerHTML = 'Canned responses ';
 
-    var editButton = createNodeWithClass('button', 'btn-link github-canned-response-edit');
-    editButton.type = 'button';
-    editButton.innerHTML = '(edit or add new)';
-    editButton.addEventListener('click', showEditView);
+    var editLink = createNodeWithClass('a', 'github-canned-response-edit');
+    editLink.href = '#gcr-ext-editor';
```

Comment:
Facebox works by looking for an element with this selector (`href`) and copies its content into `.facebox` and reveals itself.